src/shared/lib/ExtendedRouterStore.ts

```diff
@@ -44,6 +45,15 @@ function normalizeLegacySession(sessionData:any){
     return sessionData;
 }
 
+export enum QueryParameter {
```

Comment:
i think there is at least one missing.  priority. lets discuss how to get full list